addons/mass_mailing/data/mail_data.xml

```diff
@@ -17,4 +17,273 @@
+        <field name="group_id" ref="mass_mailing.group_mass_mailing_user"/>
+        <field name="tip_description" type="html">
+            <div> Want to create workflows and customer journeys ? Use the Marketing Automation app. </div>
+            <a href="https://www.odoo.com/page/marketing-automation" target="_blank">More infos</a>
+        </field>
+    </record>
+
+    <record id="digest_tip_mass_mailing_3" model="digest.tip">
+        <field name="sequence">203</field>
+        <field name="name">Email Marketing: SMS marketing application</field>
+        <field name="group_id" ref="mass_mailing.group_mass_mailing_user"/>
+        <field name="tip_description" type="html">
+            <div> To reach customers through an another channel, try the SMS Marketing app. </div>
+            <a href="https://www.odoo.com/page/sms-marketing" target="_blank">More infos</a>
+        </field>
+    </record>
+
+    <record id="digest_tip_mass_mailing_4" model="digest.tip">
+        <field name="sequence">204</field>
+        <field name="name">Email Marketing: mailing campaign option</field>
+        <field name="group_id" ref="mass_mailing.group_mass_mailing_user"/>
+        <field name="tip_description" type="html">
+            <div> To get an overview of your marketing efforts and results, activate the Mailing Campaigns option in the Email Marketing Settings. </div>
+        </field>
+    </record>
+
+    <record id="digest_tip_mass_mailing_5" model="digest.tip">
+        <field name="sequence">205</field>
+        <field name="name">Email Marketing: create Leads/Opportunities with email answers</field>
+        <field name="group_id" ref="mass_mailing.group_mass_mailing_user"/>
+        <field name="tip_description" type="html">
+            <div> If you set the alias of a Sales Team as a default reply-to, all answers to the mailing will create Leads/Opportunities. </div>
+        </field>
+    </record>
+
+    <record id="digest_tip_mass_mailing_6" model="digest.tip">
+        <field name="sequence">206</field>
+        <field name="name">Email Marketing: duplicate mailing</field>
+        <field name="group_id" ref="mass_mailing.group_mass_mailing_user"/>
+        <field name="tip_description" type="html">
+            <div> Want to re-use a mailing that worked well ? Just duplicate it and you are all set. </div>
+        </field>
+    </record>
+
+    <!-- DIGEST MAIL TEMPLATE -->
+    <record id="mail_template_mass_mailing_kpi" model="mail.template">
+        <field name="name">Email Marketing: mailing statistics</field>
+        <field name="model_id" ref="mass_mailing.model_mailing_mailing"/>
+        <field name="auto_delete" eval="True" />
+        <field name="subject">24H Stats of mailing "${object.subject}"</field>
+        <field name="email_from">${user.email}</field>
+        <field name="lang">${user.lang}</field>
+        <field name="body_html"><![CDATA[
```

Comment:
Don't use CDATA !

Reply:
@tde-banana-odoo But all mail templates use it, and without it, the XML can't be parsed

(Even with super simple content)
```xml
<record id="mail_template_mass_mailing_kpi" model="mail.template">
        <field name="name">Email Marketing: mailing statistics</field>
        <field name="model_id" ref="mass_mailing.model_mailing_mailing"></field>
        <field name="auto_delete" eval="True"></field>
        <field name="subject">24H Stats of mailing "${object.subject}"</field>
        <field name="email_from">${user.email}</field>
        <field name="lang">${user.lang}</field>
        <field name="body_html">
            <div style="max-width: 600px; margin: auto;"></div>
        </field>
    </record>
```

```
Traceback (most recent call last):
  File "/odoo/odoo/http.py", line 638, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/odoo/odoo/http.py", line 319, in _handle_exception
    raise exception.with_traceback(None) from new_cause
AssertionError: Element odoo has extra content: data, line 2
```

Reply:
I think I'll need to write all html tag like this, if I want to remove CDATA :/
```
&lt;div&gt;
```

Reply:
type="html" is your friend :) (normally most templates use type=html, cdata is old school)

Reply:
oh nice :) thanks :)